_posts/en/posts/2017-02-24-release-0.14.0.md

```diff
@@ -0,0 +1,134 @@
+
+- **[Faster new block validation and relay][faster-relay]:** miners in particular will benefit from improvements made in four existing Bitcoin Core features.  The signature cache has been upgraded for machines with many CPU cores---a test on a system with 16 cores shows a 40% speed increase in processing a new block.  The BIP152 CompactBlocks implementation will now relay some blocks before they've been fully validated, allowing those blocks to propagate across the peer-to-peer (P2P) network faster than before.  The P2P network code has also generally been refactored to allow multiple actions to happen at the same time (concurrency) as well as to increase throughput, eliminating many potential delays in processing new blocks.  Finally, unconfirmed transactions in each node's mempool can now be saved to and restored from disk when the node is restarted.
+  
+- **[Optional fee bumping][fee-bump]**: Bitcoin Core's wallet now allows users to optionally send transactions that opt-in to fee-based transaction replacement.  This allows users to increase the fee their transactions pay even after they've broadcast an earlier version of the transaction.  This feature is disabled by default.  
+
+## Improved Initial Block Download (IBD) Performance {#ibd}
+
+Over time, the constantly-increasing size of the block chain has forced new first-time nodes to process larger and large amounts of data before they can be used with a wallet to receive and send payments.  Many previous Bitcoin and Bitcoin Core releases have included major improvements designed to eliminate the pain of this Initial Block Download (IBD), also called the initial sync.  For example:
+
+| Release    | Major improvements in IBD |
+|------------|---------------------------|
+| [0.5.0][]  | Skip verification of historic (checkpointed) signatures
+| [0.8.0][]  | Switch to LevelDB & parallel signature validation
+| [0.10.0][] | Headers-first sync and parallel block download 
+| [0.11.0][] | Optional block file pruning to save disk space
+| [0.12.0][] | New fast signature validation library written from scratch (libsecp256k1)
+| [0.13.1][] | Segwit to allow skipping download of historic signatures in the future
+
+This 0.14.0 release includes two more improvements that significantly increase IBD speed:
+
+1. **Assumed valid blocks** takes the optimization introduced in Bitcoin 0.5.0 of skipping verification of historic signatures and separates it from the checkpoints mechanism that was introduced in Bitcoin 0.3.2 to prevent denial-of-service attacks during IBD.  Assumed valid blocks allows users to tell Bitcoin Core about a block they consider to be valid so that Bitcoin Core can skip verifying signatures in preceding blocks. Bitcoin Core 0.14.0 also ships with a default valid block ([block FIXME:height][], FIXME:date), which the user can either disable or change.  ([Learn more][assumed valid blocks])
```

Comment:
I think it's worth mentioning that this is an improvement over checkpoints because it doesn't force a node onto a particular chain, rather, _if_ a node is checking a block that is an ancestor of a known-good one, script checking can be skipped.

That's nice because it allows for developers to assert a fact in code (block X and its ancestors are valid), rather than stating "we strongly believe that Y is the valid chain with the most work at this time"